can-bus/json/Debug/Debug_alerts.json

```
@@ -1,4 +1,3 @@
 {
-    "msg_id": 700,
-    "alerts": []
+    
 }
```

nit: missing blank line
```suggestion
}

```